test/prob/loglogistic/loglogistic_test.hpp

```diff
@@ -0,0 +1,80 @@
+// Arguments: Doubles, Doubles, Doubles
+#include <stan/math/prim.hpp>
+
+using stan::math::var;
+using std::numeric_limits;
+using std::vector;
+
+class AgradDistributionsLoglogistic : public AgradDistributionTest {
+ public:
+  void valid_values(vector<vector<double> >& parameters,
+                    vector<double>& log_prob) {
+    vector<double> param(3);
+
+    param[0] = 1.0;  // y
+    param[1] = 2.0;  // alpha
+    param[2] = 3.0;  // beta
+    parameters.push_back(param);
+    log_prob.push_back(-1.21639532432449293253);  // expected log_prob
+
+    param[0] = 1.4;  // y
+    param[1] = 1.6;  // alpha
+    param[2] = 2.4;  // beta
+    parameters.push_back(param);
+    log_prob.push_back(-0.87286484103664152556);  // expected log_prob
+  }
+
+  void invalid_values(vector<size_t>& index, vector<double>& value) {
+    // y
+
```

Comment:
Non-positive y is also invalid, right? There should be at least one test for invalid y if it exists.

Reply:
Yes, I missed that, thanks for pointing it out! I'll add the suggested test.